submissions/Valkirin/pop-up/index.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Pup-up without JS</title>
+    <link rel="stylesheet" href="source.css" />
+  </head>
+  <body>
+    <header class="header">
+      <nav class="nav">
+        <ul class="menu list">
+          <li class="nav_item">
+            <a href="#" class="link">Green</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="link">Peace</a>
+          </li>
+          <li class="nav_item">
+            <input type="checkbox" id="popup" class="popUpControl" />
+            <label for="popup" class="button"></label>
+            <div class="box">
+              <ul class="array list">
+                <li class="content_item">
+                  <a href="#" class="link">
+                    <span class="icon">
+                      <img src="ico/kid.svg" alt="Kid" />
+                    </span>
+                    <span class="icon_text">Kid</span>
+                  </a>
+                </li>
+                <li class="content_item">
+                  <a href="#" class="link">
+                    <span class="icon">
+                      <img src="ico/store.svg" alt="Store" />
+                    </span>
+                    <span class="icon_text">Store</span>
+                  </a>
+                </li>
+                <li class="content_item">
+                  <a href="#" class="link">
+                    <span class="icon">
+                      <img src="ico/discount.svg" alt="Discount" />
+                    </span>
+                    <span class="icon_text">Discount</span>
+                  </a>
+                </li>
+                <input type="checkbox" id="more" class="moreControl" />
+                <label for="more" class="moreButton">Show more</label>
+                <hr class="row" />
```

Comment:
not valid place for elements different from `li` as a children of `ul`. More info:  [validator results](https://validator.w3.org/nu/?doc=https%3A%2F%2Fvalkirin.github.io%2Fpop-up%2F&showsource=yes&showoutline=yes). You can add `input` & `label` directly above  the `ul`. And wrap `hr` into `li` if you can't find good css solution to separate blocks visually.